### DISCUSSION TITLE
People Search *{searchterm}*

### DISCUSSION THREAD

**Discussion by Silas7100** [microsoft-search/pnp-modern-search#3231] [Q&A] [answered]
Hello,
i have created a People Search and in the query template i use the query: {searchterm}*
This works perfectly, when i search for "Andr", i get also "Andre".
Is it also possible to get the result, when i search for "ndre"?

I tried it with *{searchterm}*, but that doesn't work.

**Comment by wobba** [comment, marked as answer]
No, as wildcards only work at the end of a term.
